source/extensions/filters/http/ext_proc/mutation_utils.h

```diff
@@ -37,8 +37,13 @@ class MutationUtils : public Logger::Loggable<Logger::Id::filter> {
   static void applyBodyMutations(const envoy::service::ext_proc::v3alpha::BodyMutation& mutation,
                                  Buffer::Instance& buffer);
 
+  // Determine if a particular HTTP status code is valid.
+  static bool isValidHttpStatus(int code);
+
 private:
-  static bool isSettableHeader(absl::string_view key, bool replacing_message);
+  static bool isSettableHeader(const envoy::config::core::v3::HeaderValueOption& header,
+                               bool replacing_message);
+  static bool isAppendableHeader(absl::string_view key);
```

Comment:
Consider splitting the code changes to a separate PR (they probably should also have tests which we don't want to be inside this PR).

Reply:
This was an accidental include, it has been removed